src/components/Answers/Answers.tsx

```diff
@@ -1,11 +1,57 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import './Answers.css';
 
-interface IProps {
-  answer: string | number
+interface Iprops{
+  questions: string
+  guess: string | number
+  currentCountry: {
+    name: string
+    population: number
+    capital: string
+    borders: string[]
+  } 
 }
 
-const Answers:React.FC<IProps> = ({answer}) => {
+const Answers: React.FC<Iprops> = ({ currentCountry, questions, guess }) => {
+
+  const checkPopulation = () => {
+    if (guess < currentCountry.population + 100000 || guess > currentCountry.population - 100000) {
```

Comment:
I think this is a very user-friendly addition to the logic! It will allow users to not have to guess a super specific number